### DISCUSSION TITLE
Some of the Rosetta processes crashed!

### DISCUSSION THREAD

**Discussion by Abdelzain** [RosettaCommons/rosetta#265] [ROSIE]
Every time I run protein-protein docking this message appears to me after two hours of (running) status. I don't know if this means the docking process doesn't complete or what is the error I should avoid 

any advice please

**Comment by roccomoretti** [comment]
That's a somewhat generic error message -- it basically means that something has gone wrong with the underlying Rosetta runs. In order to properly debug it, we'd need to examine the output logs (and the ROSETTA_CRASH.log) which was produced during the run.
